modules/nextflow/src/main/groovy/nextflow/container/ContainerBuilder.groovy

```diff
@@ -65,8 +67,13 @@ abstract class ContainerBuilder<V extends ContainerBuilder> {
         return (V)this
     }
 
-    V setCpus( String value ) {
-        this.cpus = value
+    V setCpus(String value) {
+        this.cpus = String.format("%.1f", Float.parseFloat(value))
+        return (V)this
+    }
```

Comment:
I think this should take a Float and just assign to `this.cpus`

Reply:
Whoops, I reverted this change earlier by mistake along with other style changes.

@pditommaso , shall we work on an `editorconfig` next?

Reply:
No problem. I think I can generate the editor config with Intellij, let me check.

Reply:
That'd be really helpful :+1:

Reply:
@pditommaso, here's a plugin which seems to do what we want 
https://plugins.jetbrains.com/plugin/7294-editorconfig

Reply:
Nice!